internal/mode/static/state/change_processor.go

```diff
@@ -243,3 +244,9 @@ func (c *ChangeProcessorImpl) Process() (bool, *graph.Graph) {
 
 	return true, c.latestGraph
 }
+
+func (c *ChangeProcessorImpl) GetLatestGraph() *graph.Graph {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+	return c.latestGraph
```

Comment:
```suggestion
	defer c.lock.Unlock()

	return c.latestGraph
```

nit: we usually like a newline after defer statements